frontend/views/containers/payments/MonthOverview.vue

```diff
@@ -36,14 +36,20 @@ import { mapGetters } from 'vuex'
 import { PAYMENT_NOT_RECEIVED } from '@model/contracts/shared/payments/index.js'
 import ProgressBar from '@components/graphs/Progress.vue'
 import { L } from '@common/common.js'
-import { humanDate } from '@model/contracts/shared/time.js'
+import { addTimeToDate, humanDate } from '@model/contracts/shared/time.js'
 
 export default ({
   name: 'MonthOverview',
   components: {
     ProgressBar
   },
   methods: {
+    getDueDate () {
+      return addTimeToDate(this.getStartDate(), this.groupSettings.distributionPeriodLength)
+    },
+    getStartDate () {
+      return this.periodStampGivenDate(new Date())
+    },
```

Comment:
Is this `currentPaymentPeriod`?

Reply:
@snowteamer Alex is right, because Vue.js is not going to automatically update the UI when `new Date()` moves us into a new period. This is for 2 reasons:

1. This is a method, not a computed property
2. Even if it were a computed property, it still wouldn't work because computed properties are only updated in response to reactive changes. Since `new Date()` is not a reactive property itself, and is not updated as a result of a reactive Vue.js-related change (as `reactiveDate` is), the computed property doesn't get updated either because that code is never re-run